ploigos-tool-autogov/Containerfile.ubi8

```
@@ -0,0 +1,54 @@
+ARG BASE_IMAGE=quay.io/ploigos/ploigos-base:latest.ubi8
+
+FROM $BASE_IMAGE
+ARG PLOIGOS_USER_UID
+
+# labels
+ENV DESCRIPTION="Ploigos tool container with Rekor."
+LABEL \
+    maintainer="Ploigos <ploigos@redhat.com>" \
+    name="ploigos/ploigos-tool-openscap" \
+    summary="$DESCRIPTION" \
+    description="$DESCRIPTION" \
+    License="GPLv2+" \
+    architecture="x86_64" \
+    io.k8s.display-name="Ploigos - Tool - Rekor" \
+    io.k8s.description="$DESCRIPTION" \
+    io.openshift.expose-services="" \
+    io.openshift.tags="ploigos,rekor" \
+    com.redhat.component="ploigos-tool-rekor-container"
```

The component should reference `ploigos-tool-autogov`?

```suggestion
    com.redhat.component="ploigos-tool-autogov"
```